src/pilot/assign_role.py

```diff
@@ -125,7 +121,7 @@ def parse_arguments():
                         metavar="ROLE")
     parser.add_argument("-f",
                         "--flavor-settings",
-                        default="~/pilot/flavors_settings.json",
+                        default="flavors_settings.json",
```

Comment:
I don't think we should be changing the defaults or behavior for flavor_settings.json as part of this patch

Reply:
Okay. Replace as is